react/11/3/src/guess.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+export default function Guess(props) {
+  const num = Math.floor(Math.random() * 1000);
+
+  console.log(num);
+
+  function checkGuess() {
```

Comment:
Your solution works, but you are not actually using React here.

In React you almost never access the DOM directly. To change the DOM you need to put a state/prop into it `<div>{text}</div>` and then change that state/prop. So if `text` was a state you would have to call `setText("...` somewhere.

For example, the message can be done in the following way:
```js
const [message, setMessage] = useState("put in a guess")
...
let msg;

if (Number(guess.value) > num) {
  msg = getMessage("too big", "too small");
} else if (Number(guess.value) < num) {
  msg = getMessage("too small", "too big");
} else {
  msg = "Boom !!";
}

setMessage(msg)
```